### DISCUSSION TITLE
add isAfterMonth isAfterDay isBeforeMonth isBeforeDay

### DISCUSSION THREAD

**Discussion by utamori** [jogboms/time.dart#68] [Ideas]
DateTIme class has `isAfter` `isBefore`  
but There is no API to determine whether a given date is after or before the day after or before the day before.
also month

```dart
final DateTime specificDate = DateTime(2021, 01, 01);
final DateTime sameDate = DateTime(2021, 01, 01).add(const Duration(seconds: 5));
final DateTime otherDate = DateTime(2021, 01, 02);

print(specificDate.isAfterDay(sameDate)); // false
print(specificDate.isAfterDay(otherDate)); // true
```

**Comment by FMorschel** [comment]
Hi! I'm a little bit confused by your phrase but I do believe I understood your request. I'm not sure what does @jogboms thinks about adding this to the package.

But I have probably some good news for you. I'm the owner of the [due_date package](https://github.com/FMorschel/due_date/) and I'm currently working on a new class called `Period`. And this class is the abstraction of a period of time. As of right now, I have some implementations and I think they should help you if you still need this. I intend to release a new version of the package this weekend with this class and some related changes.

**Reply by FMorschel** [reply]
My package is finally updated with the Period classes that I wanted. I took a lot of time to review it because of personal things, but now it's published and you can check it out!

**Reply by jogboms** [reply]
Thanks @FMorschel 
Good work!
